apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/TableQuickstart/QuickstartAIWidget.tsx

```diff
@@ -0,0 +1,190 @@
+  const [aiPrompt, setAiPrompt] = useState('')
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const {
+    generateTables,
+    isGenerating,
+    error: apiError,
+    tables,
+    clearTables,
+  } = useAITableGeneration()
+
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.focus()
+    }
+  }, [])
+
+  const handleSelectTemplate = useCallback(
+    (template: TableSuggestion) => {
+      const tableField = convertTableSuggestionToTableField(template)
+      onSelectTable(tableField)
+      toast.success(`Applied ${template.tableName} template. You can customize the fields below.`, {
+        duration: SUCCESS_MESSAGE_DURATION_MS,
+      })
+    },
+    [onSelectTable]
+  )
+
+  const handleGenerateTables = useCallback(
+    async (promptOverride?: string) => {
+      const promptToUse = promptOverride ?? aiPrompt
+      if (!promptToUse.trim() || isGenerating) return
+
+      await generateTables(promptToUse)
+    },
+    [aiPrompt, generateTables, isGenerating]
+  )
+
+  const handleQuickIdea = useCallback(
+    (idea: string) => {
+      setAiPrompt(idea)
+      handleGenerateTables(idea)
+    },
+    [handleGenerateTables]
+  )
+
+  return (
+    <div className="rounded-lg border border-default bg-surface-100 p-6 transition-all hover:border-foreground-muted">
+      <div className="mb-4">
+        <div className="flex items-center gap-2 mb-2">
+          <AiIconAnimation size={16} loading={false} />
+          <h3 className="text-base font-medium">Generate tables with AI</h3>
+        </div>
+        <p className="text-sm text-foreground-lighter">
+          Describe your app and AI will create a complete table schema.
+        </p>
+      </div>
+
+      <div className="space-y-3">
+        <div className="relative">
+          <Input
+            ref={inputRef}
+            placeholder="Describe your app (e.g., 'recipe sharing app')"
+            value={aiPrompt}
+            onChange={(e) => setAiPrompt(e.target.value)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter' && !e.shiftKey) {
+                e.preventDefault()
+                if (aiPrompt.trim()) {
+                  handleGenerateTables()
+                }
+              }
+            }}
+            disabled={isGenerating || disabled}
+            aria-label="Table description for AI generation"
+            className="pr-24"
+          />
+          <Button
+            type="default"
+            size="tiny"
+            disabled={!aiPrompt.trim() || isGenerating || disabled}
+            onClick={() => handleGenerateTables()}
+            icon={<AiIconAnimation loading={isGenerating} />}
+            className="absolute right-1 top-1/2 -translate-y-1/2 !space-x-1"
+          >
+            {isGenerating ? 'Generating...' : 'Generate'}
+          </Button>
+        </div>
+
+        {apiError && (
+          <div className="text-sm text-red-600" role="alert">
```

Comment:
We can leave the `role="alert"` as we have no better mechanism to handle this right now, but dynamically adding `role="alert"` typically doesn't work for screen readers -- it's more robust to have a static alert element that's always on the page, and to populate it with new messages as needed